pkg/rbd/rbd_util.go

```diff
@@ -47,9 +47,9 @@ const (
 )
 
 type rbdVolumeOptions struct {
-	VolName              string            `json:"volName"`
-	Monitors             string            `json:"monitors"`
-	Pool                 string            `json:"pool"`
+	VolName  string `json:"volName"`
+	Monitors string `json:"monitors"`
+	Pool     string `json:"pool"`
 	AdminSecretName      string            `json:"adminSecret"`
 	AdminSecretNamespace string            `json:"adminSecretNamespace"`
 	AdminID              string            `json:"adminID"`
```

Comment:
since secret are passed through csi volume user credentials, we don't need to persist them any more

Reply:
Agree 100%, I have not removed them because api changes are still not confirmed.